ios/brave-ios/Sources/Shared/Extensions/URLExtensions.swift

```diff
@@ -355,6 +355,49 @@ extension URL {
     return urlHost == "talk.brave.com"
   }
 
+  // Check if the website is a night mode blocked site
+  public var isNightModeBlockedURL: Bool {
```

Comment:
Can we introduce unit tests for this? Would be good to always verify that we are explicitly blocking host matches correctly

Reply:
Also just FYI, this is not the right place for this kind of list. Its a Brave client specific feature, not a generic URL extension

Reply:
Fixed. https://github.com/brave/brave-core/pull/27798/commits/64738e0f451c0a0d8795fc2b7467af7cce2fcbe7

Reply:
Didn't fix second comment